index.bs

```diff
@@ -527,8 +527,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
     |extension| → |argument| of <code>|options|.{{MakeCredentialOptions/extensions}}</code>:
     1. If |extension| is not supported by this client platform, then [=continue=].
 
-    1. Otherwise, let |result| be the result of running |extension|'s [=client processing=] algorithm on |argument|. If the
-        algorithm returned an error, [=continue=].
+    1. Otherwise, let |result| be the result of running |extension|'s [=client extension processing=] algorithm on |argument|. If the
+        algorithm returned an error, return [=a promise rejected with=] either that error or a
```

Comment:
Since this aspect of the PR still seems to be generating discussion, I'll plan to remove this part and file it as a separate issue, letting us continue discussing it, while applying the changes in the PR that are non-controversial clear improvements over what's there.